master/getting-started/kubernetes/upgrade.md

```diff
@@ -117,7 +100,7 @@ kubectl uncordon node-01
 ```
 
 
-> **Note**: You may want to pre-fetch new Docker image to ensure the new 
+> **Note**: You may want to pre-fetch new Docker image to ensure the new
```

Comment:
You are definitely missing an article here, but I can't tell whether you need an "a" or a "the" (before "Docker")

Reply:
I guess this is what I get for making my text editor automatically remove trailing spaces :p